extension/src/shared/view-layers/rootLayer.ts

```diff
@@ -135,7 +135,7 @@ const createViewLayerWithComponentGroup = (
       shouldInitialize,
     ),
     id: datasetId,
-    format: data?.format ? REEARTH_DATA_FORMATS[data.format] : undefined,
+    format: data?.format ? REEARTH_DATA_FORMATS[data.format] : "gtfs",
```

Comment:
This is for test. I'll remove this before merging. We can't test usecase data for now, so I added this hack.